src/processor.ts

```diff
@@ -54,18 +60,53 @@ processor.run(new TypeormDatabase(), async ctx => {
                             nonce,
                             result,
                             blockNumber,
-                            timestamp: new Date(timestamp)
+                            timestamp,
                         }));
                     }
 
                 }
             }
+            else if (item.kind === "event") {
+                if (WHITELIST_CONFIG.Events.includes(item.event.name)) {
+                    let recipient = item.event.args.who;
+                    // in bridge transfers, worker accounts can't be the final recipient of the index business
```

Comment:
Exactly, so I guess we also need handle this on solver when produce the solution